tests/h/search/query_test.py

```diff
@@ -165,6 +165,81 @@ def search(self, search, pyramid_request):
 
 
 class TestAnyMatcher(object):
+    def test_matches_uriparts(self, search, Annotation):
+        Annotation(target_uri="bar.com")
+        matched_ids = [Annotation(target_uri="foo.com").id,
+                       Annotation(target_uri="foo/bar.com").id]
+
+        result = search.run({"any": "foo"})
+
+        assert sorted(result.annotation_ids) == sorted(matched_ids)
+
+    def test_matches_quote(self, search, Annotation):
+        Annotation(target_selectors=[{'exact': 'selected bar text'}])
+        matched_ids = [Annotation(target_selectors=[{'exact': 'selected foo text'}]).id,
+                       Annotation(target_selectors=[{'exact': 'selected foo bar text'}]).id]
+
+        result = search.run({"any": "foo"})
+
+        assert sorted(result.annotation_ids) == sorted(matched_ids)
+
+    def test_matches_text(self, search, Annotation):
+        Annotation(text="bar is best")
+        matched_ids = [Annotation(text="foo is fun").id,
+                       Annotation(text="foo is bar's friend").id]
+
+        result = search.run({"any": "foo"})
+
+        assert sorted(result.annotation_ids) == sorted(matched_ids)
+
+    def test_matches_tags(self, search, Annotation):
+        Annotation(tags=["bar"])
+        matched_ids = [Annotation(tags=["foo"]).id,
+                       Annotation(tags=["foo", "bar"]).id]
+
+        result = search.run({"any": "foo"})
+
+        assert sorted(result.annotation_ids) == sorted(matched_ids)
+
+    def test_matches_all_fields(self, search, Annotation):
```

Comment:
This test is already covered by the above tests for individual fields, is it not?

Reply:
I think here I was going for, lets make sure they match any of the fields at the same time but I think this is also covered by the or test below so I'll just remove it. I agree it's a bit repetitive.